cmd/filters/add_test.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2024 The K8sGPT Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package filters
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/fatih/color"
+	"github.com/k8sgpt-ai/k8sgpt/pkg/analyzer"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAddCmd(t *testing.T) {
+	require.Equal(t, "add", addCmd.Name())
+
+	err := addCmd.Args(&cobra.Command{}, []string{"arg1"})
+	require.NoError(t, err)
+
+	err = addCmd.Args(&cobra.Command{}, []string{"arg1", "arg2"})
```

Comment:
what are these arg1, arg2 values? what will actual values for these?

Reply:
I thought about it and realized that these are useless, hence, I've got rid of them in the latest version.